src/dymaptic.GeoBlazor.Core/Scripts/arcGisJsInterop.ts

```diff
@@ -1926,7 +1926,7 @@ async function createWidget(widget: any, viewId: string): Promise<Widget | null>
             });
 
             copyValuesIfExists(widget, search, 'activeMenu', 'activeSourceIndex', 'allPlaceholder',
-                'autoSelect', 'disabled', 'includeDefaultSources', 'label', 'locationEnabled', 'maxResults',
```

Comment:
why is the label removed?

Reply:
It was redundant, it's on line 2325 for all widgets.